.github/workflows/pr-check-vale.yaml

```diff
@@ -0,0 +1,39 @@
+---
+#
+# Copyright (C) 2023 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Validate website language on pull request
+
+on:
+  pull_request:
+    paths:
+      - 'website/**'
+
+jobs:
+  vale:
+    name: runner / vale
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: errata-ai/vale-action@715c322f56159d8efe0f2efbae10aba6f4e16d93
```

Comment:
Would be good to add the tag as comment otherwise hard to read and maintain

See https://github.com/redhat-developer/intellij-openshift-connector/blob/main/.github/workflows/ci.yml#L67